core/ide/src/main/java/org/alice/ide/croquet/models/ast/keyed/KeyedBlank.java

```diff
@@ -92,6 +89,11 @@ private KeyedBlank(ArgumentListProperty<JavaKeyedArgument> argumentListProperty)
 
   @Override
   protected void updateChildren(List<CascadeBlankChild> children, BlankNode<JavaKeyedArgument> blankNode) {
+    // For clever reasons, this code has no idea what is in the scene, or what any of the methods can do.
+    // For this method (e.g. move or say), find the arguments it could have (e.g. duration, text, animation style)
+    // Once we know what the options are, we can to check if it already has a value (and thus is handled elsewhere)
```

Comment:
Extra "to" in "… we can to check…".

Reply:
ah, good catch. i re-wrote this one a few too many times